types/react-native-honeywell-scanner/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for react-native-honeywell-scanner 1.0
+// Project: https://github.com/Volst/react-native-honeywell-scanner
+// Definitions by: Adam Walker <https://github.com/crashdoom>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.4
+
+export namespace HoneywellScanner {
```

Comment:
Instead of exporting the namespace, you want to add `export default HoneywellScanner` instead since usage shows that it is imported as default. Refering https://github.com/Volst/react-native-honeywell-scanner#usage

Reply:
Oops! Pushed a fix.